html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="index.html">
+    <div>
+      <label for="email">Email</label><br />
```

Comment:
Instead of using a `<br>` it is better to use a `<div>` You can wrap the input, for example:

```HTML
<div>
  <label for="email">Email</label>
  <div>
    <input name="email" type="email" id="email">
  </div>
</div>
```